types/node/crypto.d.ts

```diff
@@ -164,7 +154,13 @@ declare module "crypto" {
         getPublicKey(encoding: HexBase64Latin1Encoding): string;
         getPrivateKey(): Buffer;
         getPrivateKey(encoding: HexBase64Latin1Encoding): string;
+        /**
```

Comment:
according to docs `DiffieHellman.setPrivateKey` and `DiffieHellman.setPublicKey` are not deprecated but `ecdh.setPublicKey` is.

Reply:
Oh, we don't have that, so I'm just going to ignore it.